doc/usermanual/darkroom/modules/correction/liquify.xml

```diff
@@ -220,14 +220,14 @@
         </informaltable>
       </para>
       <para>
-        It is possible to add a control point on a line by ctrl-click on a segment. You may
-        remove a control point from a line by ctrl-right- click on the node center.
+        It is possible to add a control point on a line by Ctrl-click on a segment. You may
+        remove a control point from a line by Ctrl-right-click on the node center.
```

Comment:
Spaces and `+`

Reply:
I'm not sure how to spell it. Suggestion: Ctr + right-click